.travis.yml

```diff
@@ -8,7 +8,6 @@ env:
 matrix:
   include:
   - env: export PYTHON=3.7
-  - env: export PYTHON=3.6
```

Comment:
What is the motivation to remove python36? In travis, build are run in parallel, so you will not save time.
I don't know any python library, which doesn't support at least the two last minor python version.

Reply:
Philosophical more than practical on this one. If the Python3.6 build fails and the 3.7 passes my advice would be to stop building on 3.6 - when that's the attitude it's hard to say that we support 3.6 at all. Perhaps in the future (when we are depended on by other repos and need to be a bit more configurable) this will make sense but at present isolate pyxem only environments are to be encouraged. Also appveyor doesn't build in parallel, which is the slower part of the suite, though I concede this is a lame excuse.

Reply:
I don't see too much reason to remove python3.6 right now and don't particularly agree that we want to encourage "isolated pyxem only environments" someone might be working on developing something with a new dependency, or working with another package for some part of their analysis and that's all up to them. I think we just want to be as convenient as possible and support 3.6 is certainly not less convenient for others - right now it also isn't particularly inconvenient for us. If it becomes very inconvenient for some reason then that would be different of course.